helm-chart/test.values.yaml

```diff
@@ -14,14 +14,23 @@ cvat:
           value: cvat.settings.testing_rest
     worker:
       import:
+        replicas: 1
+        additionalEnv:
+        - name: NUMPROCS
+          value: '2'
         additionalVolumeMounts:
         - mountPath: /home/django/share
           name: cvat-backend-data
           subPath: share
+      export:
+        replicas: 1
+        additionalEnv:
+        - name: NUMPROCS
+          value: '2'
     utils:
-      additionalEnv:
-        - name: DJANGO_SETTINGS_MODULE
-          value: cvat.settings.testing_rest
```

Comment:
If I remember correctly, [these](https://github.dev/cvat-ai/cvat/blob/1e7ff33d70b341538072184d152235fe3de5c43d/tests/python/rest_api/test_tasks.py#L5062-L5150) tests were skipped because they were unstable.